docs/user-guides/client-api-config.md

```diff
@@ -0,0 +1,676 @@
+    // Represent the filter target configuration.
+    repeated Target targets = 1;
+  }
+}
+```
+
+#### timestamp
+
+`timestamp` is the timestamp when the vector is inserted.
+When `timestamp` is not set, the current time will be used.
+
+## Update Service
+
+`Update` is updating vectors already inserted in the `vald-agent` component.
+It requires the new vector, its ID (the target ID already indexed), and optional configuration.
+
+### Configuration
+
+```rpc
+// Represent update configuration.
+message Config {
+  // Check whether or not the same set of vector and ID is already inserted.
+  bool skip_strict_exist_check = 1;
+  // Configuration for filters if your Vald cluster uses filters.
+  Filter.Config filters = 2;
+  // The timestamp when the vector was inserted.
+  int64 timestamp = 3;
+}
+```
+
+<details><summary>Update Configuration Sample (Go)</summary><br>
+
+```go
+package main
+
+import (
+	"context"
+	"time"
+
+	"github.com/vdaas/vald-client-go/v1/payload"
+	"github.com/vdaas/vald-client-go/v1/vald"
+	"google.golang.org/grpc"
+)
+
+func example() {
+	// Create connection
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	target := "localhost:8080"
+	conn, err := grpc.DialContext(ctx, target)
+	if err != nil {
+		panic(err)
+	}
+	// Init Vald client
+	client := vald.NewValdClient(conn)
+
+	// Update sample
+	location, err := client.Update(ctx, &payload.Update_Request{
+		// Vector information (mandatory)
+		Vector: &payload.Object_Vector{
+			// Set the vector
+			Vector: []float32{0, 1, 2},
+			// Set the specific ID for the vector
+                        // The ID must be already indexed.
+			Id:     "sample",
+		},
+		// Update configuration (optional)
+		Config: &payload.Update_Config{
+			SkipStrictExistCheck: false,
+			Filters: &payload.Filter_Config{
+				Targets: []*payload.Filter_Target{
+					{
+						Host: "vald-ingress-filter",
+						Port: 8081,
+					},
+				},
+			},
+			Timestamp: time.Now().UnixMilli(),
+		},
+	})
+	if err != nil {
+		panic(err)
+	}
+	...
+}
+```
+
+</details>
+
+#### skip_strict_exist_check
+
+`skip_strict_exist_check` (default value is `false`) is a flag to skip checking whether the same set of the vector and ID is already inserted or not.
+If it is set as `true`, the checking function will be skipped.<BR>
+
+When `skip_strict_exist_check` is `false`, the following checking steps will run in the update process:
+
+1. Check whether the set of ID and vector is already inserted.
+   If there is no data, the update process ends with returning the `NOT_FOUND` error.
+1. After passing the step.1, check whether the request vector is the same as the indexed vector.
+   If it is the same, the update process ends with returning the `ALREADY_EXIST` error.
+
+The update process will continue if all of the above steps have been passed.
+
+#### filters
+
+`filters` is the configuration when using filter functions.
+In the `Update` section, it is popular to use ingress filtering.
```

Comment:
```suggestion
In the `Update` section, it is popular to use ingress filtering when applying the filter's configuration.
```

Reply:
same as here, it is better to use `common` instead of `popular` here.
https://github.com/vdaas/vald/pull/1783/files#r963211146